integration-tests/src/test/java/com/datastax/oss/quarkus/tests/CassandraMetricsIT.java

```diff
@@ -18,24 +18,23 @@
 import static io.restassured.RestAssured.when;
 import static org.hamcrest.core.StringContains.containsString;
 
-import com.datastax.oss.quarkus.runtime.api.session.QuarkusCqlSession;
 import com.datastax.oss.quarkus.test.CassandraTestResource;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusTest;
-import javax.inject.Inject;
+import javax.ws.rs.core.Response;
 import javax.ws.rs.core.Response.Status;
 import org.junit.jupiter.api.Test;
 
 @QuarkusTest
 @QuarkusTestResource(CassandraTestResource.class)
 public class CassandraMetricsIT {
 
-  @Inject QuarkusCqlSession cqlSession;
-
   @Test
   public void should_report_driver_metrics_via_metrics_endpoint() {
-    // when
-    cqlSession.execute("select *  from system.local");
+
+    /* Trigger some CQL activity via the test REST endpoint.  We're not particularly
+     * interested in the product that gets created here, only that it was created. */
+    when().post("/cassandra/product/test1").then().statusCode(Response.Status.OK.getStatusCode());
```

Comment:
Nit: can we change the product id to `metrics-test` or something similar? When running many ITs in a row it's easy to have side-effects when creating items with similar ids.

Reply:
Good idea, I like this... fix coming up